configs/components/rubygem-semantic_puppet.rb

```diff
@@ -1,25 +1,8 @@
 component "rubygem-semantic_puppet" do |pkg, settings, platform|
-  pkg.version "0.1.2"
-  pkg.md5sum "192ae7729997cb5d5364f64b99b13121"
-  pkg.url "https://rubygems.org/downloads/semantic_puppet-#{pkg.get_version}.gem"
-
-  pkg.build_requires "ruby"
-
-  # Because we are cross-compiling on sparc, we can't use the rubygems we just built.
-  # Instead we use the host gem installation and override GEM_HOME. Yay?
-  pkg.environment "GEM_HOME" => settings[:gem_home]
-
-  if platform.is_windows?
-    pkg.environment "PATH" => "$$(cygpath -u #{settings[:gcc_bindir]}):$$(cygpath -u #{settings[:ruby_bindir]}):$$(cygpath -u #{settings[:bindir]}):/cygdrive/c/Windows/system32:/cygdrive/c/Windows:/cygdrive/c/Windows/System32/WindowsPowerShell/v1.0"
-  end
-
-  # PA-25 in order to install gems in a cross-compiled environment we need to
-  # set RUBYLIB to include puppet and hiera, so that their gemspecs can resolve
-  # hiera/version and puppet/version requires. Without this the gem install
-  # will fail by blowing out the stack.
-  pkg.environment "RUBYLIB" => "#{settings[:ruby_vendordir]}:$$RUBYLIB"
+  pkg.url "git@github.com:puppetlabs/semantic_puppet.git"
+  pkg.ref "v0.1.2"
 
   pkg.install do
-    ["#{settings[:gem_install]} semantic_puppet-#{pkg.get_version}.gem"]
+    ["cp -pr lib/* #{settings[:ruby_vendordir]}"]
```

Comment:
That's a great question. This makes it available to puppetserver. But if we want *both* those things (puppetserver availability and visibility to rubygems) then we need a new mechanism somehow.

Meanwhile, it seems like more of the discussion is going on in RE-7610, so maybe let's follow along there and leave this PR as a POC for what one approach looks like.